### PR TITLE
Add simple landing page

### PR DESCRIPTION
**Pull request by GRardB**

### CODE REVIEW

**Comment by GRardB** [comment]
[Create a super basic landing page](https://trello.com/c/Mtu8uYm3/21-create-a-super-basic-landing-page)

File: src/components/Layouts/Main/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    background-color: ${lightenColor(Colors.Blue, 0.2)};
+  }
+`
+export const Navbar = () => (
+  <Container>
+    <UINavbar>
+      <Brand>TubeTracker</Brand>
+      <NavLinks>
+        <SignupLink to="/signup">Sign up</SignupLink>
+        <LoginLink to="/login">Log in</LoginLink>
```
Comment (malinatran) [inline src/components/Layouts/Main/Navbar.tsx]: Minor thing here, but do we use single quotes for import statements and double quotes for everything else?

File: src/components/Layouts/Main/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { Navbar } from './Navbar'
+
+export const MainLayout: React.SFC = ({ children }) => (
+  <>
```
Comment (malinatran) [inline src/components/Layouts/Main/index.tsx]: This convention for fragments is cool!

File: src/components/Layouts/Main/Navbar.tsx (hunk 1 above)
Comment (GRardB) [reply src/components/Layouts/Main/Navbar.tsx]: Yeah, Prettier is set to use single quotes, but that only applies to non-JSX stuff. I think by default, JSX is set to use double quotes, which is typically what you see in HTML.

File: src/components/Layouts/Main/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import { Colors, lightenColor } from 'common/colors'
+import { Container } from '../../Container'
+
+const UINavbar = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
```
Comment (malinatran) [inline src/components/Layouts/Main/Navbar.tsx]: Do we want to use a base size variable?

Comment (GRardB) [reply src/components/Layouts/Main/Navbar.tsx]: Probably not. We don't have a design system or anything. I think it's overkill for this project.

**Pull request merged**